cfg/params.yaml

```diff
@@ -326,9 +326,9 @@
   usage: >-
     Specifies an alternative custom endpoint for fetching data. The custom endpoint
     must support the equivalent resources and operations as the GCS JSON endpoint,
-    https://storage.googleapis.com/storage/v1. If a custom endpoint is not
+    storage.<UNIVERSE_DOMAIN>:443. If a custom endpoint is not
```

Comment:
Do we have any public doc around universe domain? Linking here would be good for customer.

Reply:
No we have internal documentation only that too universe specific like GCS, TPC.